packages/protocol/test/governance/validators.ts

```diff
@@ -2059,4 +2059,35 @@ contract('Validators', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#forceDeaffiliateIfValidator', async () => {
+    const validator = accounts[0]
+    const group = accounts[1]
+
+    beforeEach(async () => {
+      await registerValidator(validator)
+      await registerValidatorGroup(group)
+      await validators.affiliate(group)
+    })
+
+    describe('when the caller is the DowntimeSlasher Address', async () => {
+      beforeEach(async () => {
+        await registry.setAddressFor(CeloContractName.DowntimeSlasher, validator)
+        await registry.setAddressFor(CeloContractName.DoubleSigningSlasher, accounts[3])
+        await registry.setAddressFor(CeloContractName.Governance, accounts[4])
+      })
+
+      it('should succeed when sender is the downtime slasher contract', async () => {
```

Comment:
Add tests that it should succeed when called by governance, doublesigningslasher, e.g. `await validators.forceDeafiliateIfValidator(validator, { from: accounts[3] })`